### ISSUE TITLE
using babel@6

### ISSUE THREAD

**Issue by mastilver** [jarsbe/react-simple#6]
I know babel-browser have been deprecated: https://babeljs.io/docs/usage/browser/

But is there any hack / workaround to make it work?

**Comment by jackcallister** [comment]
Hey sorry for the late reply on this. Have been thinking a lot about this and need a second opinion. Perhaps bypassing ES6 and JSX altogether is the best way to keep this simple. I'm completely comfortable bypassing ES6 for simplicity but think JSXless approach makes things a bit more complicated. What do you think? 

Take a look at this PR #6 for reference.

**Comment by jackcallister** [comment]
Also knocking out babel fixes issue #2 as the site can then be served from the filesystem.

**Comment by battaile** [comment]
> JSXless approach makes things a bit more complicated

I agree with this. 

I also feel like JSX is something which should be included given getting up and running with React as a goal, since I think its a reasonable assumption for someone to make that JSX would be the default in such a project.  I can see that being debatable though.

**Comment by jackcallister** [comment]
Yeah, I really want JSX in there - a shame there isn't a standalone browser side jsx transpiler anymore...

**Comment by mastilver** [comment]
This gist by @RickWong could interest you: https://gist.github.com/RickWong/dd2a0aba86699ded4cc295d84601d540
